@here/harp-omv-datasource/lib/OmvDataSource.ts

```diff
@@ -273,6 +292,18 @@ export class OmvDataSource extends TileDataSource<OmvTile> {
         }
     }
 
+    /** @override */
+    setPoliticalView(politicalView?: string): void {
+        if (this.m_decoderOptions.politicalView !== politicalView) {
+            this.m_decoderOptions.politicalView = politicalView;
+            this.configureDecoder(undefined, undefined, undefined, {
+                filterDescription: this.m_decoderOptions.filterDescription,
+                featureModifiers: this.m_decoderOptions.featureModifiers,
+                politicalView
```

Comment:
`undefined` will not be transfered, so "unset doesn't work" here.

Yes, i KNOW `null` would work here, but we just need some workaround like `"neutral"` text and OmvDecoder should be able to intrpret it.

Reply:
Ok solved with empty string for default point of view - no OmvPoliticalViewFeatureModifier.